packages/node/src/lib/process-request.ts

```diff
@@ -178,7 +178,8 @@ export default function processRequest(
   const host = fixHeader(req.headers['x-forwarded-host']) || req.headers.host;
   // We use a fake host here because we rely on the host header which could be redacted.
   // We only ever use this reqUrl with the fake hostname for the pathname and querystring.
-  const reqUrl = new URL(req.url, `https://readme.io`);
+  // req.originalUrl is express specific, req.url is node.js
+  const reqUrl = new URL(req.originalUrl || req.url, `https://readme.io`);
```

Comment:
see previous express questions

Reply:
ugh, this is why: https://github.com/readmeio/metrics-sdks/blob/31e7f34c5c07fb9a62f2599b0dd9c82fbbdde7dc/packages/node/src/lib/express-middleware.ts#L106-L108 ok yeah this is a non trivial thing to resolve.